addons/account/models/partner.py

```diff
@@ -538,6 +538,11 @@ def _get_company_currency(self):
     duplicated_bank_account_partners_count = fields.Integer(
         compute='_compute_duplicated_bank_account_partners_count',
     )
+    is_coa_installed = fields.Boolean(default=lambda partner: bool(partner.env.company.chart_template_id), compute="_compute_is_coa_installed")
+
+    def _compute_is_coa_installed(self):
+        for partner in self:
+            partner.is_coa_installed = bool(self.env.company.chart_template_id)
```

Comment:
```suggestion
    is_coa_installed = fields.Boolean(default=lambda partner: bool(partner.env.company.chart_template_id), store=False)
```
No need for the compute if the field is not stored (that was the hacky part I referred to earlier) :nerd:

Reply:
yup sorry

Reply:
> No need for the compute **_if the field is not stored_**

But you need the `store=False` otherwise it will not recompute :wink: